app/code/Magento/Catalog/Console/ImageResizeOptions.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Catalog\Console;
+
+use Symfony\Component\Console\Input\InputOption;
+
+class ImageResizeOptions
+{
+
+    /**
+     * Key for jobs option
+     */
+    const JOBS_AMOUNT = 'jobs';
+
+    /**
+     * Key for product-offset option
+     */
+    const PRODUCT_OFFSET = 'product-offset';
+
+    /**
+     * Key for product-limit option
+     */
+    const PRODUCT_LIMIT = 'product-limit';
+
+    /**
+     * Default jobs amount
+     */
+    const DEFAULT_JOBS_AMOUNT = 0;
+
+    /**
+     * Default jobs amount
```

Comment:
This docBlock looks like an artifact of copy-pasting. Please review and update descriptions accordingly.